doc/Learn/Markup/toc.yml

```diff
@@ -15,6 +15,8 @@
       href: xref:Uno.Extensions.Markup.StaticAndThemeResources
     - name: Using Uno.Themes.WinUI.Markup
       href: xref:Uno.Extensions.Markup.UnoThemes
+    - name: Source and Relative Source
+      href: xref:Uno.Extensions.Markup.SourceUsage
```

Comment:
@eriklimakc https://github.com/unoplatform/uno.extensions/pull/2287 is now merged so you can rebase your branch with the latest as this fix is already done. Thanks